<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>647</width>
    <height>418</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="mainWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="mainLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetMaximumSize</enum>
      </property>
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <property name="spacing">
       <number>20</number>
      </property>
      <item row="0" column="0" rowspan="5">
       <widget class="QScrollArea" name="imageArea">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>100</height>
         </size>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="content">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>287</width>
           <height>378</height>
          </rect>
         </property>
        </widget>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QGroupBox" name="scaleBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>179</width>
          <height>50</height>
         </size>
        </property>
        <property name="title">
         <string>Zoom</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <layout class="QHBoxLayout" name="scaleLayout">
           <item alignment="Qt::AlignRight">
            <widget class="QRadioButton" name="radioButton1x">
             <property name="minimumSize">
              <size>
               <width>35</width>
               <height>17</height>
              </size>
             </property>
             <property name="text">
              <string>1x</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item alignment="Qt::AlignHCenter">
            <widget class="QRadioButton" name="radioButton2x">
             <property name="minimumSize">
              <size>
               <width>35</width>
               <height>17</height>
              </size>
             </property>
             <property name="text">
              <string>2x</string>
             </property>
            </widget>
           </item>
           <item alignment="Qt::AlignHCenter">
            <widget class="QRadioButton" name="radioButton3x">
             <property name="minimumSize">
              <size>
               <width>45</width>
               <height>17</height>
              </size>
             </property>
             <property name="text">
              <string>3x</string>
             </property>
            </widget>
           </item>
           <item alignment="Qt::AlignHCenter">
            <widget class="QRadioButton" name="radioButton4x">
             <property name="minimumSize">
              <size>
               <width>45</width>
               <height>17</height>
              </size>
             </property>
             <property name="text">
              <string>4x</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item row="3" column="1" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
       <widget class="QGroupBox" name="gammaBox">
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>100</height>
         </size>
        </property>
        <property name="title">
         <string>Gamma correction</string>
        </property>
        <widget class="QWidget" name="verticalLayoutWidget">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>281</width>
           <height>61</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gammaLayout">
          <item row="1" column="0">
           <widget class="QSlider" name="gammaSlider">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <property name="minimum">
             <number>10</number>
            </property>
            <property name="maximum">
             <number>40</number>
            </property>
            <property name="value">
             <number>10</number>
            </property>
            <property name="tracking">
             <bool>true</bool>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="tickPosition">
             <enum>QSlider::TicksBothSides</enum>
            </property>
            <property name="tickInterval">
             <number>1</number>
            </property>
           </widget>
          </item>
          <item row="0" column="0" alignment="Qt::AlignLeft|Qt::AlignVCenter">
           <widget class="QLineEdit" name="gammaField">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>20</height>
             </size>
            </property>
            <property name="cursorPosition">
             <number>0</number>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item row="2" column="1" alignment="Qt::AlignVCenter">
       <widget class="QGroupBox" name="standartBox">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>50</height>
         </size>
        </property>
        <property name="title">
         <string>Standart</string>
        </property>
        <widget class="QWidget" name="layout">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>271</width>
           <height>20</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="standartlayout">
          <item>
           <widget class="QRadioButton" name="radioButtonBT601">
            <property name="text">
             <string>BT.601</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButtonBT709">
            <property name="text">
             <string>BT.709</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QGroupBox" name="unpackingBox">
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>80</height>
         </size>
        </property>
        <property name="title">
         <string>Unpacking</string>
        </property>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>275</width>
           <height>53</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="unpackingLayout">
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="typeLayout">
            <item>
             <widget class="QRadioButton" name="type1">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>1 macropixel</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="type2">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>2 macropixels</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="type3">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>3 macropixels</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <layout class="QHBoxLayout" name="coeffLayout">
            <item>
             <layout class="QHBoxLayout" name="coeff1Layout">
              <item alignment="Qt::AlignHCenter">
               <widget class="QLabel" name="coeff1Label">
                <property name="text">
                 <string>Ratio</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="coeff1spinBox">
                <property name="minimum">
                 <number>1</number>
                </property>
                <property name="maximum">
                 <number>25</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="coeff2Layout">
              <item alignment="Qt::AlignHCenter">
               <widget class="QLabel" name="label">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>4</width>
                  <height>22</height>
                 </size>
                </property>
                <property name="text">
                 <string>:</string>
                </property>
               </widget>
              </item>
              <item alignment="Qt::AlignHCenter">
               <widget class="QSpinBox" name="coeff2spinBox">
                <property name="minimum">
                 <number>1</number>
                </property>
                <property name="maximum">
                 <number>25</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="coeff3Layout">
              <item alignment="Qt::AlignHCenter">
               <widget class="QLabel" name="coeff3Label">
                <property name="text">
                 <string>:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="coeff3spinBox">
                <property name="minimum">
                 <number>1</number>
                </property>
                <property name="maximum">
                 <number>25</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="filesBox"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
